docs/authentication/personal-access-tokens.md

```diff
@@ -1,26 +1,50 @@
-# Personal Access Tokens
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
 
-With these changes, we introduced a way to generate a "Personal Access Token" suitable for programmatic use with both the DataHub GraphQL
-and DataHub Rest.li (Ingestion) APIs.
+# About DataHub Personal Access Tokens
 
-Personal Access Tokens have a finite lifespan (default 3 months) and currently cannot be revoked without changing the signing key that
-DataHub uses to generate these tokens (via the TokenService described above). Most importantly, they inherit the permissions
-granted to the user who generates them.
+<FeatureAvailability/>
 
-## Generating Personal Access Tokens
+Personal Access Tokens or PATs for short, allow users to represent themselves in code and programmatically use DataHub's APIs in deployments where security is a concern. 
 
-To generate a personal access token, users must have been granted the "Generate Personal Access Tokens" (GENERATE_PERSONAL_ACCESS_TOKENS) or "Manage All Access Tokens" Privilege via a [DataHub Policy](../authorization/policies.md). Once
-they have this permission, users can navigate to **'Settings'** > **'Access Tokens'** > **'Generate Personal Access Token'** to generate a token.
+Used along-side with [authentication-enabled metadata service](introducing-metadata-service-authentication.md), PATs add a layer of protection to DataHub where only authorized users are able to perform actions in an automated way.
 
-![](../imgs/generate-personal-access-token.png)
+## Personal Access Tokens Setup, Prerequisites, and Permissions
 
-The token expiration dictates how long the token will be valid for. We recommend setting the shortest duration possible, as tokens are not currently
-revokable once granted (without changing the signing key).
+To use PATs, two things are required:
+ 1. Metadata Authentication must have been enabled in GMS. See `Configuring Metadata Service Authentication` in [authentication-enabled metadata service](introducing-metadata-service-authentication.md).
+ 2. Users must have been granted the `Generate Personal Access Tokens` or `Manage All Access Tokens` Privilege via a [DataHub Policy](../authorization/policies.md).
 
+Once configured, users should be able to navigate to **'Settings'** > **'Access Tokens'** > **'Generate Personal Access Token'** to generate a token:
+
+![](../imgs/pats/generate-personal-access-token.png)
+
+If you have configured permissions correctly the `Generate new token` should be clickable.
+
+:::note
+
+If you see `Token based authentication is currently disabled. Contact your DataHub administrator to enable this feature.` then you must enable authentication in the metadata service (step 1 of the prerequisites).
+
+:::
+
+## Creating Personal Access Tokens
+
+Once in the Manage Access Tokens Settings Tab:
+
+1. Click `Generate new token` where a form should appear.
+
+![](../imgs/pats/token-creation-form.png)
```

Comment:
Couple of nit-picks for consistency with other guides & improved readability:

* images should be stored in https://github.com/datahub-project/static-assets
*  please use HTML to embed images so we can size down (70%) & center-align:

```
<p align="center">
  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/master/imgs/token-creation-form.png"/>
</p>
```